src/Cookie/FileCookieJar.php

```diff
@@ -88,7 +89,7 @@ public function load(string $filename): void
             foreach ($data as $cookie) {
                 $this->setCookie(new SetCookie($cookie));
             }
-        } elseif (\strlen($data)) {
+        } elseif (is_scalar($data) && $data != '') {
```

Comment:
Is this better?
```suggestion
        } elseif (is_scalar($data) && $data !== '') {
```

Reply:
IMHO I don't see the point of `is_scalar`.. IIRC the `jsoDecode` will return a scalar value. This is a PHPStan issue that should understand that `all` scalar values are returned by `jsonDecode`

Reply:
`jsonDecode` returns an array from perspective of the `FileCookieJar`, several other values are under test coverage.

What is asserted here, is whether any other non-empty value throws the `RuntimeException`. 

I've changed the loose `$data != ''` to `!empty($data)` to clarify that in a826a3af692aef8ebe680584f9a434efdada896d.